contracts/oracle/src/lib.rs

```diff
@@ -333,6 +333,68 @@ impl Contract {
         Ok(())
     }
 
+    #[payable]
+    #[handle_result]
+    pub fn admin_mint_kyc(
+        &mut self,
+        mint_data: Vec<(AccountId, u64)>,
+        memo: Option<String>,
+    ) -> Result<Promise, CtrError> {
+        self.assert_admin();
+
+        let num_tokens = mint_data.len();
+        let storage_deposit = mint_deposit(num_tokens);
+        require!(
+            env::attached_deposit() >= storage_deposit,
+            format!(
+                "Requires attached deposit at least {} yoctoNEAR",
+                storage_deposit
+            )
+        );
+
+        let now: u64 = env::block_timestamp_ms();
+        let mut tokens_metadata: Vec<(AccountId, Vec<TokenMetadata>)> = Vec::new();
```

Comment:
we can create a vector with predetermined capacity

Reply:
done